components/UiKit/Adapt.tsx

```diff
@@ -20,9 +20,23 @@ const Adapt = styled.div<{ $mt?: boolean; $width650?: boolean }>`
     padding-right: 15px;
   }
 
-  ${({ $mt }) => $mt && `margin-top: 60px;`};
+  ${({ $mt }) =>
+    $mt &&
+    `
+    margin-top: 80px;
+
+    @media ${device.M} {
+      margin-top: 60px;
+    }
+    
+    @media ${device.S} {
+      margin-top: 40px;
+    }
+  `};
 
   ${({ $width650 }) => $width650 && `max-width: 650px`};
+
+  ${({ $width714 }) => $width714 && `max-width: 650px`};
```

Comment:
Should it be like this?
```suggestion
  ${({ $width714 }) => $width714 && `max-width: 714px`};
```